build/yamls/flow-visibility/flow-visibility.yaml

```diff
@@ -0,0 +1,418 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app: flow-visibility
+  name: flow-visibility
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: flow-visibility
+  name: flow-visibility
+  namespace: flow-visibility
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app: flow-visibility
+  name: grafana-role
+  namespace: flow-visibility
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app: flow-visibility
+  name: grafana-role-binding
+  namespace: flow-visibility
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: grafana-role
+subjects:
+  - kind: ServiceAccount
+    name: grafana
```

Comment:
I'm confusing with the ServiceAccount here. It seems we only have a ServiceAccount named `flow-visibility` created above, but we bind the role to a ServiceAccount called `grafana`. Besides, it seems neither of the two ServiceAccounts is used?

Reply:
You are right. `flow-visibility` Service Account is not useful in our case. Removed it. Thanks!

Reply:
Just to check if the `grafana` ServiceAccount is used? I see some rules are bound to this ServiceAccount, but I do not find where it is created or assigned to pods in this file? Is it used in other places?

Reply:
I am not sure what was the intention here, but I believe @yanjunz97 is correct: we need to define the service account, and also use it in the Grafana deployment spec.